native/src/components/TtsContainer.tsx

```diff
@@ -52,14 +54,16 @@ const TtsContainer = ({ children }: TtsContainerProps): ReactElement => {
     })
   }, [])
 
-  const enabled = Boolean(buildConfig().featureFlags.tts) && !unsupportedLanguagesForTts.includes(languageCode)
+  const enabled = buildConfig().featureFlags.tts && !unsupportedLanguagesForTts.includes(languageCode)
+  const isContentAvailable = Array.isArray(sentences) && sentences.length > 0
```

Comment:
:upside_down_face: Makes things clearer imo as otherwise I am wondering why it would not be an array.
```suggestion
  const isContentAvailable = sentences !== null && sentences.length > 0
```

Reply:
if it's not enabled it will show the snackbar ...

Reply:
Sorry, I don't understand, can you rephrase please?

Reply:
I got confused by my own code 😅 NVM